python/ray/_private/gcs_pubsub.py

```diff
@@ -14,6 +13,7 @@
 
 import ray._private.gcs_utils as gcs_utils
 import ray._private.logging_utils as logging_utils
+from ray._raylet import Config
```

Comment:
This would make autoscaler and dashboard depend on raylet transitively. Is it ok?

Reply:
actually, not raylet, but `_raylet.so`. here we are only using the flags, so it should be ok. basically, it's only about where do we interpret the flags and the only thing is about the default value.
previously we have code in cpp/python to interpret this. now we put everything in cpp.

Reply:
Isn't this opening `_raylet.so`?

Reply:
yes, it is. any concerns?

Reply:
I thought you mentioned that autoscaler & dashboard do not want to depend on this. Can we double check with the owners? Btw why is consolidating to the C++ feature flag necessary in the first place?

Reply:
it's much easier to maintain the logic I think. basically, right now, there are two places interpreting this, to enable the flag by default you have to change two places. why not then?

They have `_raylet.so` always in their library, I think. And if they `import ray.x.y.z` it's hard to exclude `_raylet.so` from loading since `ray.x.y.z` might use `_raylet.so`.

Reply:
one thing is about the protocol change which is not an issue here. for example, autoscaler is 1.x version and ray is 1.y version. this flag is always self-contained no protocol involved.